prow/config/jobs/istio-1.12.yaml

```diff
@@ -314,7 +314,7 @@ jobs:
   - presubmit_optional
   - presubmit_skipped
   - hidden
-  name: integ-pilot-istiodless-multicluster-tests
+  name: integ-pilot-less-mc-tests
```

Comment:
would be nice to keep "istiodless" as "less" confused me. I think removing the `-tests` should give us room.